spec/game_team_spec.rb

```diff
@@ -133,126 +133,6 @@
       end
     end
 
-    describe "::team_id_and_shots - helper" do
```

Comment:
Did you accidentally delete all of these tests? Or am I reading this wrong?

Reply:
I have all of those tests on my main branch. Maybe I created the branch before pulling down main. I'm checking if it's possible to resolve in GitHub.